### PR TITLE
{data}[foss/2021a] redis-py v4.3.1 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3702.slaking.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/66ccbdbec0be3bd0639f5bd3ba5a10b5 for a full test report.

File: easybuild/easyconfigs/r/redis-py/redis-py-4.3.1-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+version = "4.3.1"
+
+homepage = "https://github.com/redis/redis-py"
+description = "The Python interface to the Redis key-value store."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('typing-extensions', '3.10.0.0'),
```
Comment (verdurin) [inline easybuild/easyconfigs/r/redis-py/redis-py-4.3.1-foss-2021a.eb:13]: Would it make sense to add `Redis` itself as a dependency here?

**Review by boegel — APPROVED** [review]
lgtm

**Comment by verdurin** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**